ui/partials/perf/alertsctrl.html

```diff
@@ -68,33 +68,36 @@
         <div ng-show="!user.is_staff" style="padding: 6px;">
           {{alertSummary.getStatusText()}}
         </div>
```

Comment:
We should delete this div block (the alerts view now looks wrong when not logged in as a sheriff)

![screen shot 2016-07-07 at 10 46 26 am](https://cloud.githubusercontent.com/assets/20569/16657341/273025fe-4430-11e6-8e88-e3746a0cd3f3.png)